lombok-modules/lombok-3/README.md

```diff
@@ -0,0 +1,8 @@
+## Project Lombok
+
+This module contains articles about Project Lombok.
+
+### Relevant Articles:
+
+- [Lombok's Constructor Annotations] (https://www.baeldung.com/difference-between-allargsconstructor-requiredargsconstructor-and-noargconstructor-in-lombok)
```

Comment:
```suggestion
```

We leave the current article out of the README.md since there is a separate process that populates these post-publish, when the URL of the article has been finalized